src/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java

```diff
@@ -222,7 +222,7 @@ public boolean requestHasToken(String reqBody) {
 		return this.getTokens(msg.getRequestBody().toString(), msg.getRequestHeader().getURI().toString());
 	}
 	
-	private List<AntiCsrfToken> getTokens(String reqBody, String targetUrl) {
+	public List<AntiCsrfToken> getTokens(String reqBody, String targetUrl) {
```

Comment:
I was thinking more about this, it might be better to just use `getTokens(HttpMessage)` (with `requestMessage`) instead of making public. This allow us to use the same method for JSON messages (and others, which we can tell by its `Content-Type`) once the support is added.